recipes/calceph/all/test_package/conanfile.py

```diff
@@ -3,8 +3,17 @@
 
 
 class TestPackageConan(ConanFile):
-    settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build_requirements(self):
+        if self.settings.os == "Macos" and self.settings.arch == "armv8":
```

Comment:
so, after the relocatable thing, all test packages will have to add it for M1, right?

Reply:
I guess. Surprisingly, most CMake based recipes don't need it in test package.

Reply:
okay, maybe it's time to think about it again...
with that, we may more often hit:
1. options bug with different options in requirements vs build_requirements (OpenSSL)
2. epoch computation bug, when requirements are added in test_package

maybe M1 environment should provide a more recent cmake out of the box? /cc @jgsogo